apps/studio/src/components/CmsSidebar/CmsSideNav.tsx

```diff
@@ -120,11 +120,19 @@
           return (
             <>
               {/* NOTE: required for focus ring */}
-              <Box p="4px">
+              <Box m="4px">
                 <AccordionButton
                   boxSizing="border-box"
                   disabled={isDisabled}
                   borderRadius="0.25rem"
+                  _focusVisible={{
+                    boxShadow: "none !important",
+                    outline: `2px solid var(--chakra-colors-utility-focus-default)`,
+                    outlineOffset: "0.125rem",
+                    _dark: {
+                      outline: `2px solid var(--chakra-colors-utility-focus-inverse)`,
+                    },
+                  }}
                   _hover={{
                     backgroundColor: "interaction.muted.main.hover",
                   }}
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +145]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->lint error here
```
Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator
```<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/opengovsg/isomer/437?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->